consensus/qbft/src/integration-test/java/org/hyperledger/besu/consensus/qbft/support/TestContextBuilder.java

```diff
@@ -263,10 +277,19 @@ public TestContext build() {
     if (genesisFile.isPresent()) {
       try {
         final GenesisState genesisState = createGenesisBlock(genesisFile.get());
+        Block genesisBlock = genesisState.getBlock();
```

Comment:
Could just set the base fee in the genesis block of the genesis file instead of overriding it here

Reply:
I avoided setting it in the genesis since that's not how zeroBaseFee config works (it kicks in post-genesis for the next block you create). The only impact this has is on the error you get when the transaction simulator configuration fix isn't in place.

However, I've just realised there's an edge case if you have a network with validator contract at genesis: we attempt to access the contract _before_ we create the next block and so don't "zeroBaseFee" doesn't get a chance to set the baseFee to zero. So in reality, I should get the same error as for "zeroBaseFee: false".

So I will remove this whole block of code in TestContextBuilder.

Reply:
removed